scripts/cibuild

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${CI}" ]]; then
+    set -x
+fi
+
+set -u
```

Comment:
Scripts to Rule Them All 😁! Let's break down what some of this boilerplate does:

- `set -e` - Exit immediately if anything fails.
- `set -x` - Print a trace of every command.
- `set -u` - Exit if any variables are unset.